app/lib/test_applications.rb

```diff
@@ -223,9 +223,9 @@ def create_application_to_courses(
         @application_form.application_choices.each do |choice|
           choice.update_columns(
             sent_to_provider_at: time,
```

Comment:
Might be nice to audit this anyway for visibility, but up to you

Reply:
I actually started with that but realised we aren't auditing any other changes we update with time so would be inconsistent. Might need to revisit this whole class and possibly wrap things in transactions, and add appropriate audit comments etc.